test/test.c

```diff
@@ -41,14 +41,29 @@ UBENCH(c, do_nothing) {
   UBENCH_DO_NOTHING(b);
 }
 
+UBENCH_EX(c, ex)
+{
+  int b[1024];
+  int i;
+  int sum;
+  memset(b, 0x0, sizeof(b));
+
+  UBENCH_KEEP_RUNNING()
+  {
+    sum = 0;
+    for(i = 0; i < 1024; ++i)
+      sum += i;
+  }
+}
+
 struct c_my_fixture {
   char *data;
 };
 
 UBENCH_F_SETUP(c_my_fixture) {
   const int size = 128 * 1024 * 1024;
   ubench_fixture->data = (char *)malloc(size);
-  memset(ubench_fixture->data, ' ', size - 2);
+  memset(ubench_fixture->data, ' ', size - 1);
```

Comment:
Is this an independent bug you fixed?!

Reply:
It is, I'll make a separate PR for this. Maybe i should cancel this and make a few clean ones?

Reply:
Naw don't make another PR - I was just curious if I had made a mistake!